android/sdk/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Jitsi Meet SDK</string>
+    <string name="dropbox_app_key"></string>
+    <string name="ongoing_notification_title">Текущая встреча</string>
+    <string name="ongoing_notification_text">Нажмите, чтобы вернуться к встрече.</string>
+    <string name="ongoing_notification_action_hang_up">Отключиться</string>
+    <string name="ongoing_notification_action_mute">Отключить звук</string>
+    <string name="ongoing_notification_action_unmute">Включить звук</string>
+    <string name="ongoing_notification_channel_name">Ongoing Conference Notifications</string>
+</resources>
```

Comment:
@saghul do we add translations to the android resource values?

Reply:
No, but it doesn't hurt either.